djweb/dj_comp_hist/models.py

```diff
@@ -44,8 +44,8 @@ class Folder(models.Model):
     full = models.CharField(max_length=191)
     number = models.IntegerField(default=0)
 
-    # def __str__(self):
-    #     return self.full
+    def __str__(self):
```

Comment:
can we define also `def __repr__` to return something something like `return f"<Folder {self.full} - {self.number}>"`  then when you see it you'll know it's a folder object and not get into the problem we saw before with full = blank.